doc/nrf/releases/release-notes-2.1.1.rst

```diff
@@ -0,0 +1,102 @@
+.. _ncs_release_notes_211:
+
+|NCS| v2.1.1 Release Notes
+##########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+|NCS| delivers reference software and supporting libraries for developing low-power wireless applications with Nordic Semiconductor products in the nRF52, nRF53, nRF70, and nRF91 Series.
+The SDK includes open source projects (TF-M, MCUboot, OpenThread, Matter, and the Zephyr RTOS), which are continuously integrated and redistributed with the SDK.
+
+Release notes might refer to "experimental" support for features, which indicates that the feature is incomplete in functionality or verification, and can be expected to change in future releases.
+To learn more, see :ref:`software_maturity`.
+
+Highlights
+**********
+
+This minor release adds the following changes on top of the :ref:`nRF Connect SDK v2.1.0 <ncs_release_notes_210>`:
+
+* :ref:`Support for production <software_maturity>` for Matter SDK v1.0.
```

Comment:
This would be changed by PMT - but just make sure that we're considering production support only for Thread - not for Wi-Fi.